big_tests/tests/service_mongoose_system_stats_SUITE.erl

```diff
@@ -59,34 +60,42 @@ init_per_group(_GroupName, Config) ->
 end_per_group(_GroupName, Config) ->
     Config.
 
-init_per_testcase(user_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
+init_per_testcase(system_stats_are_reported_to_google_analytics_when_mim_starts, Config) ->
     create_events_collection(),
-    enable_user_stats(mim()),
+    enable_system_stats(mim()),
     delete_prev_client_id(mim()),
```

Comment:
Ok, this is terribly suspicious. We're first starting the service, which creates this mnesia table, and then we _delete_ the table! Most likely the process just kept the client id cached on it's state so it never needed to check for this table, but, damn 😃 
I don't think we need to delete this table in any case. The service will create the table when it starts, and if the table was already there is because it was made on a cluster by some other node, so we also don't want to delete it. What we need is to make sure this is properly handled on source, and not force things on tests 😛

Reply:
Removed delete_prev_client_id in innit_per_test_case when gathering stats is allowed.